custom/litmus-checker/Dockerfile

```diff
@@ -14,5 +14,7 @@ LABEL maintainer="LitmusChaos"
 
 #Install kubectl
 #Copy binaries from build stage
-COPY --from=builder /output/checker /
```

Comment:
You copying to root director. Just have to change the path

Reply:
The image enters as a root user, we have to add a new user and group first and then we can just change the ownership

Reply:
The path is also change to /var/run